tests/director/go.mod

```diff
@@ -23,3 +23,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.7 // indirect
 	k8s.io/apimachinery v0.17.0 // indirect
 )
+
+replace github.com/kyma-incubator/compass/components/director => github.com/crabtree/compass/components/director v0.0.0-20200121070913-f3240c4624e9
```

Comment:
Just a reminder to delete it

Reply:
Sure, will prepare another PR after merging this one.